configuration/persistence.md

```diff
@@ -184,18 +184,28 @@ Here is the full list of available persistence extensions:
+| `<item>.changedSince(ZonedDateTime)`   | Checks if the State of the Item has (ever) changed since a certain point in time                                                                                           |
+| `<item>.changedBetween(ZonedDateTime, ZonedDateTime)`   | Checks if the State of the Item has (ever) changed between certain points in time                                                                                           |
 | `<item>.updatedSince(ZonedDateTime)`   | Checks if the state of the Item has been updated since a certain point in time                                                                                             |
+| `<item>.updatedBetween(ZonedDateTime, ZonedDateTime)`   | Checks if the state of the Item has been updated between certain points in time                                                                                             |
 | `<item>.maximumSince(ZonedDateTime)`   | Gets the maximum value of the State of a persisted Item since a certain point in time (returns HistoricItem)                                                               |
+| `<item>.maximumBetween(ZonedDateTime ZonedDateTime)`   | Gets the maximum value of the State of a persisted Item between certain points in time (returns HistoricItem)                                                               |
 | `<item>.minimumSince(ZonedDateTime)`   | Gets the minimum value of the State of a persisted Item since a certain point in time (returns HistoricItem)                                                               |
+| `<item>.minimumBetween(ZonedDateTime, ZonedDateTime)`   | Gets the minimum value of the State of a persisted Item between certain points in time (returns HistoricItem)                                                               |
 | `<item>.averageSince(ZonedDateTime)`   | Gets the average value of the State of a persisted Item since a certain point in time. This method uses a time-weighted average calculation (see example below)            |
+| `<item>.averageBetween(ZonedDateTime, ZonedDateTime)`   | Gets the average value of the State of a persisted Item betwen certain points in time. This method uses a time-weighted average calculation (see example below)            |
 | `<item>.deltaSince(ZonedDateTime)`     | Gets the difference in value of the State of a given Item since a certain point in time                                                                                    |
+| `<item>.deltaBetween(ZonedDateTime, ZonedDateTime)`     | Gets the difference in value of the State of a given Item between certain points in time                                                                                    |
 | `<item>.evolutionRate(ZonedDateTime)`  | Gets the evolution rate of the state of a given {@link Item} since a certain point in time (returns DecimalType)                                                           |
```

Comment:
```suggestion
| `<item>.evolutionRate(ZonedDateTime)`  | Gets the evolution rate of the state of a given Item since a certain point in time (returns DecimalType)                                                           |
```